tools/pip/setup.py

```diff
@@ -152,6 +152,8 @@ def has_ext_modules(self):
         package_data['mxnet'].append('mxnet/libmkldnn.so.0')
     shutil.copytree(os.path.join(CURRENT_DIR, 'mxnet-build/3rdparty/mkldnn/include'),
```

Comment:
I thought the whole mkldnn include folder should already have been included here. Should we just change this to copy from the build folder instead?

Reply:
`mkldnn_version.h` is generated and placed under `build/include` folder. Other header files are still in this folder. So we have to copy from two different places.

Reply:
I just verified that the rest of the headers are copied there too. Let's just use the folder.

Reply:
I did not see the other headers are copied in my local environment. That's why I added separately.

```
root@8c384f679438:/build/mxnet-build/3rdparty/mkldnn/build/include# ls
mkldnn_version.h
```

Reply:
```
% ls 3rdparty/mkldnn/build/install/include
i_malloc.h                mkl_blas.h                mkl_direct_blas.h         mkl_direct_call.h         mkl_direct_types.h        mkl_dnn_types.h           mkl_lapacke.h             mkl_trans.h               mkl_version.h             mkl_vml_defines.h         mkl_vml_types.h           mkl_vsl_defines.h         mkl_vsl_types.h
mkl.h                     mkl_cblas.h               mkl_direct_blas_kernels.h mkl_direct_lapack.h       mkl_dnn.h                 mkl_lapack.h              mkl_service.h             mkl_types.h               mkl_vml.h                 mkl_vml_functions.h       mkl_vsl.h                 mkl_vsl_functions.h
```
The one in the install folder has both

Reply:
ok, changed to copy from build/install/include folder.